src/hotspot/share/gc/z/zRelocationSetSelector.hpp

```diff
@@ -78,9 +78,12 @@ class ZRelocationSetSelectorStats {
 
 class ZRelocationSetSelectorGroup {
 private:
+  static constexpr int NumPartitionsShift = 11;
+  static constexpr int NPartitions = int(1) << NumPartitionsShift;
```

Comment:
Shenandoah shouldn't leak out unprefixed internal names. Let's fix that as a separate RFE.